Workbooks/SapMonitor/PrometheusHaCluster/PrometheusHaCluster.workbook

```diff
@@ -52,6 +102,50 @@
       },
       "name": "text - 2"
     },
+    {
+      "type": 11,
```

Comment:
Can you make the button invisible for now? Samie wants to release all the alert buttons at the same time. need to add a condition like the other buttons. 
Example:
```diff
{
      "type": 11,
      "content": {
        "version": "LinkItem/1.0",
        "style": "nav",
        "links": [
          {
            "id": "990307c5-35e3-4028-9e54-a96f7aba6269",
            "linkTarget": "WorkbookTemplate",
            "linkLabel": "Alerts",
            "style": "primary",
            "linkIsContextBlade": true,
            "workbookContext": {
              "componentIdSource": "workbook",
              "resourceIdsSource": "workbook",
              "templateIdSource": "static",
              "templateId": "Community-Workbooks/SapMonitor/Alerts/Alerts",
              "typeSource": "workbook",
              "gallerySource": "workbook",
              "locationSource": "default",
              "passSpecificParams": true,
              "templateParameters": [
                {
                  "name": "AlertTemplates",
                  "source": "parameter",
                  "value": "alertparameter"
                },
                {
                  "name": "ProviderType",
                  "source": "parameter",
                  "value": "param_providers_type"
                }
              ]
            }
          }
        ]
      },
+     "conditionalVisibility": {
+       "parameterName": "dummy_neverset",
+      "comparison": "isNotEqualTo"
+   },
      "name": "links - 7"
    },
```

Reply:
ConditionalVisibility added.